podaac/subsetter/subset.py

```diff
@@ -699,8 +709,27 @@ def build_cond(str_timestamp, compare):
                 timestamp = np.datetime64(timestamp) - epoch_datetime
 
         if np.issubdtype(dataset[time_var_name].dtype, np.dtype(float)):
-            start_date = np.datetime64(dataset[time_var_name].attrs['Units'][-10:])
-            timestamp = (np.datetime64(timestamp) - start_date).astype('timedelta64[s]').astype('float')
+            start_date = None
+            for i in list(dataset[time_var_name].attrs):
+                attr_string = str(dataset[time_var_name].attrs[i])
+                if 'TAI93' in attr_string:
+                    start_date = np.datetime64('1993-01-01')
+                    continue
```

Comment:
Not sure if this was intended but this loop will pick the last attribute with 'TAI93' in it's name. If it doesn't matter which one to use in the case of multiple this could be simplified to
`start_date = np.datetime64('1993-01-01') if any(['TAI93' in attribute_name for attribute_name in dataset[time_var_name].attrs]) else None`